portal-frontend/src/app/features/public/notice-of-intent/submission/pofo-details/pofo-details.component.html

```diff
@@ -28,62 +28,60 @@
     <app-no-data *ngIf="!noiSubmission.soilToPlaceAverageDepth"></app-no-data>
   </div>
 
-  <div class="full-width soil-table">
-    <div class="grid-1"></div>
-    <div class="grid-2">
-      <strong> Fill already Placed</strong>
-    </div>
-    <div class="grid-1"><strong>Volume</strong></div>
-    <div class="grid-2">
-      {{ noiSubmission.soilAlreadyPlacedVolume }}
-      <span *ngIf="noiSubmission.soilAlreadyPlacedVolume !== null">m<sup>3</sup></span>
-      <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedVolume === null"></app-no-data>
-    </div>
-    <div class="grid-1"><strong>Area</strong></div>
-    <div class="grid-2">
-      {{ noiSubmission.soilAlreadyPlacedArea }}
-      <span *ngIf="noiSubmission.soilAlreadyPlacedArea !== null">m<sup>2</sup></span>
-      <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedArea === null"></app-no-data>
-    </div>
-    <div class="grid-1"><strong>Maximum Depth</strong></div>
-    <div class="grid-2">
-      {{ noiSubmission.soilAlreadyPlacedMaximumDepth }}
-      <span *ngIf="noiSubmission.soilAlreadyPlacedMaximumDepth !== null">m</span>
-      <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedMaximumDepth === null"></app-no-data>
-    </div>
-    <div class="grid-1"><strong>Average Depth</strong></div>
-    <div class="grid-2">
-      {{ noiSubmission.soilAlreadyPlacedAverageDepth }}
-      <span *ngIf="noiSubmission.soilAlreadyPlacedAverageDepth !== null">m</span>
-      <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedAverageDepth === null"></app-no-data>
-    </div>
+  <div class="grid-1"></div>
+  <div class="grid-double">
+    <strong> Fill already Placed</strong>
+  </div>
+  <div class="grid-1"><strong>Volume</strong></div>
+  <div class="grid-double">
+    {{ noiSubmission.soilAlreadyPlacedVolume }}
+    <span *ngIf="noiSubmission.soilAlreadyPlacedVolume !== null">m<sup>3</sup></span>
+    <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedVolume === null"></app-no-data>
+  </div>
+  <div class="grid-1"><strong>Area</strong></div>
+  <div class="grid-double">
+    {{ noiSubmission.soilAlreadyPlacedArea }}
+    <span *ngIf="noiSubmission.soilAlreadyPlacedArea !== null">m<sup>2</sup></span>
+    <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedArea === null"></app-no-data>
+  </div>
+  <div class="grid-1"><strong>Maximum Depth</strong></div>
+  <div class="grid-double">
+    {{ noiSubmission.soilAlreadyPlacedMaximumDepth }}
+    <span *ngIf="noiSubmission.soilAlreadyPlacedMaximumDepth !== null">m</span>
+    <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedMaximumDepth === null"></app-no-data>
+  </div>
+  <div class="grid-1"><strong>Average Depth</strong></div>
+  <div class="grid-double">
+    {{ noiSubmission.soilAlreadyPlacedAverageDepth }}
+    <span *ngIf="noiSubmission.soilAlreadyPlacedAverageDepth !== null">m</span>
+    <app-no-data *ngIf="noiSubmission.soilAlreadyPlacedAverageDepth === null"></app-no-data>
   </div>
 
   <div class="full-width">
```

Comment:
Need to also update Project Duration for all app types to be before soil tables and in new format